packages/entities/entities-gateway-services/src/components/GatewayServiceForm.vue

```diff
@@ -660,8 +959,8 @@ const validateName = (input: string): void => {
  * If the form.fields and formFieldsOriginal are equal, always return false
  */
 const canSubmit = computed((): boolean =>
-  (isEditing.value && (JSON.stringify(form.fields) !== JSON.stringify(formFieldsOriginal))) || ((checkedGroup.value === 'url' && !!form.fields.url) ||
-  (checkedGroup.value === 'protocol' && !!form.fields.host)))
+  (isEditing.value && (JSON.stringify(form.fields) !== JSON.stringify(formFieldsOriginal))) || ((checkedGroup.value === 'url' && !!form.fields.url && !isFormValid.value) ||
+  (checkedGroup.value === 'protocol' && !!form.fields.host && !isFormValid.value)))
```

Comment:
I'm a bit confused by this logic. You're checking if the submit button is enabled. If I'm reading the logic right you're checking that they're using the `url` field, it has a value, and the form is NOT valid. Why would you enable the button if the form is not valid?

Reply:
update this computed with more readable code and comments